--- debian/rules	2015-07-02 01:52:05.397048049 +0000
+++ debian/rules_12	2015-07-02 01:52:54.221048324 +0000
@@ -116,6 +116,15 @@
 # libkstat is required for the xfs_arc plugin.
 confflags += --disable-zfs-arc
 
+#signalFx ubuntu build 1204
+confflags += \
+		--disable-lvm \
+		--disable-turbostat \
+		--disable-write_kafka \
+		--disable-ovs_events \
+		--disable-ovs_stats
+
+
 # These plugins are Linux-specific.
 ifneq (,$(filter kfreebsd-i386 kfreebsd-amd64, $(DEB_BUILD_ARCH)))
 	confflags += \
@@ -191,7 +198,6 @@
 	sed -re 's/^(Requires:.*) pthread(.*)$$/\1\2/' \
 		/usr/lib/pkgconfig/OpenIPMIpthread.pc \
 		> debian/pkgconfig/OpenIPMIpthread.pc
-	
 	chmod 755 configure
 	PKG_CONFIG_PATH="$(CURDIR)/debian/pkgconfig:$$PKG_CONFIG_PATH" \
 	./configure $(confflags) CPPFLAGS="$(CPPFLAGS)" CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)" \
@@ -321,4 +327,3 @@
 
 binary: binary-arch binary-indep
 .PHONY: build build-arch build-indep clean binary-indep binary-arch binary install-indep install-arch
-
